bridge/opencensus/internal/otel2oc/span_context_test.go

```diff
@@ -6,12 +6,25 @@ package otel2oc
 import (
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
+	"go.opencensus.io/trace/tracestate"
+
 	octrace "go.opencensus.io/trace"
 
 	"go.opentelemetry.io/otel/trace"
 )
 
 func TestSpanContextConversion(t *testing.T) {
+	tsOc, _ := tracestate.New(nil,
+		// Oc has a reverse order of TraceState entries compared to OTel
+		tracestate.Entry{Key: "key2", Value: "value2"},
+		tracestate.Entry{Key: "key1", Value: "value1"},
+	)
+	tsOtel := trace.TraceState{}
+	tsOtel, _ = tsOtel.Insert("key1", "value1")
+	tsOtel, _ = tsOtel.Insert("key2", "value2")
```

Comment:
```suggestion
	tsOc, _ := tracestate.New(nil,
		// Oc has a reverse order of TraceState entries compared to OTel
		tracestate.Entry{Key: "key1", Value: "value1"},
		tracestate.Entry{Key: "key2", Value: "value2"},
	)
	tsOtel := trace.TraceState{}
	tsOtel, _ = tsOtel.Insert("key2", "value2")
	tsOtel, _ = tsOtel.Insert("key1", "value1")

	httpFormatOc := &tracecontext.HTTPFormat{}
```

Assume the header output is `key1=value1,key2=value2`